test-types/basic-matchers.test.ts

```diff
@@ -0,0 +1,428 @@
+/**
+ * @type ./empty.d.ts
+ */
+
+import {expectError, expectType} from 'mlh-tsd';
+import {expect, jest} from '@jest/globals';
+import type {Mock} from 'jest-mock';
+
+describe('', () => {
+  it('', () => {
+    /* Corrections of previous typings */
+
+    // FIXME: TSD's limitations. Cannot run a function if it does not exist?
+    // expectError<jest.Matchers<string>>(expect('').not.not);
+    // expectError<jest.Matchers<Promise<Promise<string>>>>(
+    //   expect('').resolves.resolves,
+    // );
+
+    expectType<string>(expect('').toEqual(''));
+    expectType<Promise<Promise<string>>>(
```

Comment:
Ideally this should be something like

```suggestion
    expectType<Promise<void>>(
```

it's probably not, tho 🙂 Something to improve once these tests land!

Reply:
Nope, it doesn't work :(

Throws out errors when tests are run.